monai/transforms/spatial/array.py

```diff
@@ -2630,16 +2630,12 @@ class GridPatch(Transform):
         patch_size: size of patches to generate slices for, 0 or None selects whole dimension
         offset: offset of starting position in the array, default is 0 for each dimension.
         num_patches: number of patches to return. Defaults to None, which returns all the available patches.
-        overlap: the amount of overlap of neighboring patches in each dimension (a value between 0.0 and 1.0).
-            If only one float number is given, it will be applied to all dimensions. Defaults to 0.0.
-        sort_fn: a callable or string that defines the order of the patches to be returned. If it is a callable, it
-            will be passed directly to the `key` argument of `sorted` function. The string can be "min" or "max",
-            which are, respectively, the minimum and maximum of the sum of intensities of a patch across all dimensions
-            and channels. Also "random" creates a random order of patches.
-            By default no sorting is being done and patches are returned in a row-major order.
+        filter_high_values: if `True` the patches with highest values will be filtered, and if `False` the patches with
+            highest values will be filtered. Defaults to None, where filtering just depends on the order and not the values.
         threshold: a value to keep only the patches whose sum of intensities are less than the threshold.
             Defaults to no filtering.
-        pad_mode: refer to NumpyPadMode and PytorchPadMode. If None, no padding will be applied. Defaults to ``"constant"``.
+        return_location: if True it return the starting location of patches as well
+        pad_mode: refer to NumpyPadMode and PytorchPadMode. If None, no padding will be applied. Defaults to None.
         pad_kwargs: other arguments for the `np.pad` or `torch.pad` function.
 
     """
```

Comment:
Seems the transform only supports `numpy` backend:
```py
def create_patches(self, input_image):
    image, *_ = convert_data_type(input_image, np.ndarray)
```
Please change it to: `backend = [TransformBackends.NUMPY]`

Thanks.